getting-started-service-registry/README.adoc

```diff
@@ -0,0 +1,197 @@
+[id="chap-getting-started-service-registry"]
+= Getting Started with {product-long}
+ifdef::context[:parent-context: {context}]
+:context: getting-started-service-registry
+
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-long: Red Hat Openshift Service Registry
+:product: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://cloud.redhat.com/beta/application-services/sr/
+:property-file-name: app-services.properties
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As a developer of applications and services, you can use {product-long} to create and set up Service Registry instances and connect your applications and services to these instances. {product} is a managed cloud service that enables you to manage schema and API definitions in your applications without having to install, configure, run, and maintain your own Service Registry clusters.
+
+//For more overview information about {product}, see [variablized link to overview here https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/].
```

Comment:
SR portal doc link will be https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/